src/pikepdf/jbig2.py

```diff
@@ -14,21 +15,42 @@
 
 
 def extract_jbig2(im_obj: pikepdf.Object, globals_obj: pikepdf.Object = None) -> Image:
-    with NamedTemp() as imgfile, NamedTemp() as globalfile, NamedTemp() as outfile:
+    # On Windows platform, the NamedTemporaryFile context manager creates the
+    # files in exclusive mode. Therefore, we need to fill the data of those
+    # files, then instruct NamedTemporaryFile to not to delete them.
```

Comment:
excess `to`

Reply:
I'm not a native speaker. Please review everything and I will update as requested.

Reply:
I'm not a native speaker either ;) Anyone makes typos and that's not a problem, but when I see one I usually leave a note.
The rest of the text sounds good to me, though I only skimmed it.

Reply:
I updated the PR. Let me know when everything OK and I will squash the changes for you :)